src/types/network.ts

```diff
@@ -14,4 +18,6 @@ export const ColonyJSNetworkMapping = {
   [Network.Gnosis]: ColonyJSNetwork.Xdai,
   [Network.GnosisFork]: ColonyJSNetwork.XdaiQa,
   [Network.Ganache]: ColonyJSNetwork.Custom,
+  [Network.ArbitrumOne]: ColonyJSNetwork.ArbitrumOne,
+  [Network.ArbitrumSepolia]: ColonyJSNetwork.ArbitrumSepolia,
```

Comment:
Note that only Arbitrum is listed here due to `colonyJS` not having support for Polygon.

This is ok for now, but will need updating if that ever becomes a requirement.